tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -112,17 +112,18 @@ public:
       FetchUpdateArgs() = default;
       FetchUpdateArgs(const FetchUpdateArgs &) = delete;
       FetchUpdateArgs(FetchUpdateArgs &&fua);
-      FetchUpdateArgs(DistributionKey_t d, std::span<const AttributeKey_t> as, std::span<d_iov_t> vs,
+      FetchUpdateArgs(DistributionKey_t d, std::span<const AttributeKey_t> as, std::vector<std::vector<d_iov_t>> &&vs,
```

Comment:
See comment on line 126, but we _might_ be able to accept a single `std::span` or something that contains everything?